internal/buildscripts/packaging/choco/splunk-otel-collector/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,179 @@
+        update_registry -path "$regkey" -name "SPLUNK_ACCESS_TOKEN" -value "$SPLUNK_ACCESS_TOKEN"
+    }
+}
+catch{
+    write-host "The SPLUNK_ACCESS_TOKEN parameter is not specified."
+}
+
+try {
+    if (!$SPLUNK_REALM) {
+        $SPLUNK_REALM = Get-ItemPropertyValue -PATH "HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" -name "SPLUNK_REALM" -ErrorAction SilentlyContinue
+    }
+}
+catch {
+    $SPLUNK_REALM = "us0"
+    write-host "The SPLUNK_REALM parameter is not specified. Using default configuration."
+}
+
+try {
+    if (!$SPLUNK_INGEST_URL) {
+        $SPLUNK_INGEST_URL =  Get-ItemPropertyValue -PATH "HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" -name "SPLUNK_INGEST_URL" -ErrorAction SilentlyContinue
+    }
+}
+catch {
+    $SPLUNK_INGEST_URL = "https://ingest.$SPLUNK_REALM.signalfx.com"
+    write-host "The SPLUNK_INGEST_URL parameter is not specified. Using default configuration."
+}
+
+try {
+    if (!$SPLUNK_API_URL) {
+        $SPLUNK_API_URL = Get-ItemPropertyValue -PATH "HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" -name "SPLUNK_API_URL" -ErrorAction SilentlyContinue
+    }
+}
+catch {
+    $SPLUNK_API_URL = "https://api.$SPLUNK_REALM.signalfx.com"
+    write-host "The SPLUNK_INGEST_URL parameter is not specified. Using default configuration."
+}
+
+try {
+    if (!$SPLUNK_HEC_TOKEN) {
+        $SPLUNK_HEC_TOKEN = Get-ItemPropertyValue -PATH "HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" -name "SPLUNK_HEC_TOKEN" -ErrorAction SilentlyContinue
+    }
+}
+catch {
+    $SPLUNK_HEC_TOKEN = $SPLUNK_ACCESS_TOKEN
+    write-host "The SPLUNK_HEC_TOKEN parameter is not specified. Using default configuration."
+}
+
+try {
+    if (!$SPLUNK_HEC_URL) {
+        $SPLUNK_HEC_URL = Get-ItemPropertyValue -PATH "HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" -name "SPLUNK_HEC_URL" -ErrorAction SilentlyContinue
+    }
+}
+catch {
+    $SPLUNK_HEC_URL = "https://ingest.$SPLUNK_REALM.signalfx.com/v1/log"
+    write-host "The SPLUNK_HEC_URL parameter is not specified. Using default configuration."
+}
+
+try {
+    if (!$SPLUNK_TRACE_URL) {
+        $SPLUNK_TRACE_URL = Get-ItemPropertyValue -PATH "HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" -name "SPLUNK_TRACE_URL" -ErrorAction SilentlyContinue
+    }
+}
+catch {
+    $SPLUNK_TRACE_URL = "https://ingest.$SPLUNK_REALM.signalfx.com/v2/trace"
+    write-host "The SPLUNK_TRACE_URL parameter is not specified. Using default configuration."
+}
+
+try {
+    if (!$SPLUNK_MEMORY_TOTAL_MIB) {
+        $SPLUNK_MEMORY_TOTAL_MIB = Get-ItemPropertyValue -PATH "HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" -name "SPLUNK_MEMORY_TOTAL_MIB" -ErrorAction SilentlyContinue
+    }
+}
+catch {
+    $SPLUNK_MEMORY_TOTAL_MIB = "512"
+    write-host "The SPLUNK_MEMORY_TOTAL_MIB parameter is not specified. Using default configuration."
+}
+
+if (!$SPLUNK_BUNDLE_DIR) {

```

Comment:
Should we check if `SPLUNK_BUNDLE_DIR` is already in the registry like the other vars?

Reply:
Yes we can check it also. Let me made that change..